packages/components/src/components/ui/SquiggleErrorAlert.tsx

```diff
@@ -103,32 +103,48 @@ const ImportChain: FC<{ error: SqImportError }> = ({ error }) => {
   ) : null;
 };
 
-export const SquiggleErrorAlert: FC<Props> = ({ error }) => {
-  function errorName(): string {
-    if (error instanceof SqCompileError) {
-      return "Compile Error";
-    } else if (error instanceof SqRuntimeError) {
-      return "Runtime Error";
-    } else if (error instanceof SqImportError) {
-      return "Import Error";
-    } else {
-      return "Error";
-    }
+function errorName(error: SqError): string {
+  if (error instanceof SqImportError) {
+    return errorName(error.wrappedError());
```

Comment:
IIRC this is a fix for how we display errors from imports, unrelated to semantic analysis.